face/FindFaces/python/find_faces_in_image.ipynb

```diff
@@ -83,32 +83,22 @@
     "                mode=FindSimilarMatchMode.MATCH_FACE,\n",
     "            )\n",
     "\n",
-    "    if source_person_id and person_group_id:\n",
-    "        # Use REST API for person ID and person group ID verification\n",
-    "        headers = {\n",
-    "            'Ocp-Apim-Subscription-Key': subscription_key,\n",
-    "            'Content-Type': 'application/json',\n",
-    "            'X-MS-AZSDK-Telemetry': 'sample=find-faces-in-image'\n",
-    "        }\n",
-    "        for search_face_id in search_face_ids:\n",
-    "            body = {\n",
-    "                'personGroupId': person_group_id,\n",
-    "                'personId': source_person_id,\n",
-    "                'faceId': search_face_id,\n",
-    "            }\n",
-    "            url = endpoint + '/face/v1.0/verify'\n",
-    "            response = requests.post(url, headers=headers, json=body)\n",
-    "            response.raise_for_status()\n",
-    "            result = response.json()\n",
-    "            result['faceId'] = search_face_id\n",
-    "            similar_faces.append(result)\n",
+    "        if source_person_id and large_person_group_id:\n",
+    "            for search_face_id in search_face_ids:\n",
+    "                result = face_client.verify_from_large_person_group(\n",
+    "                    face_id=search_face_id,\n",
+    "                    large_person_group_id=large_person_group_id,\n",
+    "                    person_id=source_person_id\n",
+    "                )\n",
```

Comment:
If you wish to use the returned FaceVerificationResult object as a dict, maybe it would be clearer to explicity call its [as_dict](https://learn.microsoft.com/en-us/python/api/azure-ai-vision-face/azure.ai.vision.face.models.faceverificationresult?view=azure-python-preview#azure-ai-vision-face-models-faceverificationresult-as-dict) method.